onadata/apps/main/tests/test_google_doc.py

```diff
@@ -14,7 +17,7 @@ def test_view(self):
             )
         input_path = os.path.join(folder, "input.html")
         with open(input_path) as f:
-            input_html = f.read()
+            input_html = smart_text(f.read())
```

Comment:
When I created this branch, I was working on another branch of Python3 in parallel. 
It's only there for retro-compatibility between python3 and python2. 
In Python2: `f.read()` is `str` (or `unicode`). 
In Python3: `f.read()` is `bytes` 

I wanted to be sure to get the same type whatever the version.

> Returns a text object representing 's' -- unicode on Python 2 and 
> str on Python 3. Treats bytestrings using the 'encoding' codec.

It could (should) be removed in next phase.

Reply:
ok, thanks. i see `encoding` defaults to `utf-8`.